pkg/doc/verifiable/credential.go

```diff
@@ -562,20 +699,20 @@ func describeSchemaValidationError(result *gojsonschema.Result) string {
 }
 
 func getCredentialSchema(schema *CredentialSchema, opts *credentialOpts) (gojsonschema.JSONLoader, error) {
-	schemaLoader := defaultSchemaLoader
 	if schema != nil && !opts.disabledCustomSchema {
 		switch schema.Type {
 		case jsonSchema2018Type:
-			if customSchemaData, err := loadCredentialSchema(schema.ID, opts.schemaDownloadClient); err == nil {
-				schemaLoader = gojsonschema.NewBytesLoader(customSchemaData)
-			} else {
-				return nil, fmt.Errorf("loading custom credential schema from %s failed: %w", schema.ID, err)
+			var err error
```

Comment:
this line can (and should be eliminated if you change L707 to not embed the if condition (and instead create a line after).

Reply:
Yeah, my stupid miss.